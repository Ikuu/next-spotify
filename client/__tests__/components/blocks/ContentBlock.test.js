import React from 'react';
import { create } from 'react-test-renderer';
import ContentBlock from '../../../components/blocks/ContentBlock';

test('ContentBlock renders', () => {
  const tree = create(<ContentBlock heading={<h1>Test</h1>} />).toJSON();
  expect(tree).toMatchSnapshot();
});
