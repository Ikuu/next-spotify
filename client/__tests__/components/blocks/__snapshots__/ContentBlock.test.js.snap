// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentBlock renders 1`] = `
<div>
  <h1>
    Test
  </h1>
</div>
`;
